code-review-chat/index.ts

```diff
@@ -162,12 +162,6 @@ class CodeReviewChatAction extends Action {
 		).data;
 
 		const pr = createPRObject(prFromApi);
-		const draft = pr.draft || false;
-		if (draft) {
-			safeLog('PR is draft, ignoring');
-			return;
-		}
-
```

Comment:
Isn't this a good thing to do?

Reply:
We check for it elsewhere (https://github.com/microsoft/vscode-github-triage-actions/blob/ef7c71041207b7cfed33cd754a88e89a1d841a1d/code-review-chat/CodeReviewChat.ts#L269-L270). This prevents us from handling draft prs.